src/main/java/fi/hsl/transitdata/pubtransredisconnect/JourneyResultSetProcessor.java

```diff
@@ -0,0 +1,51 @@
+package fi.hsl.transitdata.pubtransredisconnect;
+
+import fi.hsl.common.transitdata.TransitdataProperties;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import redis.clients.jedis.Jedis;
+
+import java.sql.ResultSet;
+import java.util.HashMap;
+import java.util.Map;
+
+public class JourneyResultSetProcessor implements IResultSetProcessor {
+
+    private static final Logger log = LoggerFactory.getLogger(JourneyResultSetProcessor.class);
+
+    private Jedis jedis;
+    private int redisTTLInSeconds;
```

Comment:
These two members are duplicated in all implementing IResultSetProcessors. Perhaps change it to an abstract base class and have the members there? minor thing but reduces duplicate code.

Reply:
well it seems like those ones are the only shared stuff in the code so not sure if it's worth the effort. your call.

Reply:
I will rethink about the placement.